# The only call to pthread_create() is in Thread.hpp, which has applied
# a fix to the stack size general case. Disable the call to gnu extension
# to make it compiles.
diff --git a/one.cpp b/one.cpp
index 2e4e638..9cd20ad 100644
--- a/one.cpp
+++ b/one.cpp
@@ -2088,7 +2088,7 @@ int __cdecl _tmain(int argc, _TCHAR* argv[])
 int main(int argc,char **argv)
 #endif
 {
-#if defined(__LINUX__) && ( (!defined(__GLIBC__)) || ((__GLIBC__ >= 2) && (__GLIBC_MINOR__ >= 18)) )
+#if !defined(__ANDROID__) && defined(__LINUX__) && ( (!defined(__GLIBC__)) || ((__GLIBC__ >= 2) && (__GLIBC_MINOR__ >= 18)) )
 	// This corrects for systems with abnormally small defaults (musl) and also
 	// shrinks the stack on systems with large defaults to save a bit of memory.
 	pthread_attr_t tattr;
